hpaction/tests/test_build_hpactionvars.py

```diff
@@ -244,7 +247,7 @@ def test_sue_for_harassment_works():
     h.sue_for_harassment = None
     v = hp.HPActionVariables()
     fill_hp_action_details(v, h)
-    assert v.problem_is_urgent_tf is None
+    assert v.sue_for_harassment_tf is None
```

Comment:
Ooops, this was a typo back when I wrote it, I think... I probably had copy-pasted the "urgent and dangerous" test and forgotten to change this line.